### PR TITLE
Fix incorrect link of `URLSearchParams.toString()`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

the `window.location.search` was incorrectly linked to `HTMLAnchorElement.search`

though, both `Location.search` and `HTMLAnchorElement.search` contains a '?' while the return value of `URLSearchParams.toString()` does not

so, the fix corrected the link and added both two examples

### Motivation

Ditto.

### Additional details

https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/toString

https://developer.mozilla.org/en-US/docs/Web/API/Location/search

https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement/search

### CODE REVIEW

File: files/en-us/web/api/urlsearchparams/tostring/index.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ The **`toString()`** method of the
 URL.
 
 > **Note:** This method returns the query string without the question
-> mark. This is different from [window.location.search](/en-US/docs/Web/API/HTMLAnchorElement/search),
+> mark. This is different from [Location.search](/en-US/docs/Web/API/Location/search)
+> and [HTMLAnchorElement.search](/en-US/docs/Web/API/HTMLAnchorElement/search),
 > which includes it.
```
Comment (wbamberg) [inline files/en-us/web/api/urlsearchparams/tostring/index.md:18]:
```suggestion
> which both include the question mark.
```

(IMO it is worth spelling things out, and avoiding words like "it" which are often ambiguous.)

**Review by wbamberg — COMMENTED** [review]
Thank you @skyclouds2001 ! Just a wording suggestion.

File: files/en-us/web/api/urlsearchparams/tostring/index.md [hunk 2]
```diff
@@ -13,8 +13,9 @@ The **`toString()`** method of the
 URL.
 
 > **Note:** This method returns the query string without the question
-> mark. This is different from [window.location.search](/en-US/docs/Web/API/HTMLAnchorElement/search),
-> which includes it.
+> mark. This is different from [Location.search](/en-US/docs/Web/API/Location/search)
+> and [HTMLAnchorElement.search](/en-US/docs/Web/API/HTMLAnchorElement/search),
```
Comment (Josh-Cena) [inline files/en-us/web/api/urlsearchparams/tostring/index.md:17]:
```suggestion
> mark. This is different from [`Location.search`](/en-US/docs/Web/API/Location/search)
> and [`HTMLAnchorElement.search`](/en-US/docs/Web/API/HTMLAnchorElement/search),
```

**Review by Josh-Cena — APPROVED** [review]
Looks great. Thanks

**Comment by Josh-Cena** [comment]
BTW, maybe you want to link to https://developer.mozilla.org/en-US/docs/Web/API/URL/search as well, since that seems the most relevant? Up to you.

**Comment by skyclouds2001** [comment]
> BTW, maybe you want to link to https://developer.mozilla.org/en-US/docs/Web/API/URL/search as well, since that seems the most relevant? Up to you.

That's a good idea.

**Pull request merged**